src/jit/lsra.h

```diff
@@ -852,6 +852,7 @@ class LinearScan : public LinearScanInterface
     Interval* getIntervalForLocalVar(unsigned varIndex)
     {
         assert(varIndex < compiler->lvaTrackedCount);
+        noway_assert(localVarIntervals[varIndex] != nullptr);
         return localVarIntervals[varIndex];
```

Comment:
Does this really need to be a noway_assert? It looks like this is a hot function, and this adds code to the release build path.